site/en/docs/webstore/using_webstore_api/index.md

```diff
@@ -121,20 +150,20 @@ https://www.googleapis.com/upload/chromewebstore/v1.1/items
 
 {% Aside %}
 
-**Note**: For a full description of the insert method, see [Items:Insert][6].
+For a full description of the insert method, see [Items:Insert][cws-api-insert].
 
 {% endAside %}
 
 ### Uploading a package to update an existing store item {: #uploadexisitng }
 
 ```text
-Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$APP_ID
+Endpoint: https://www.googleapis.com/upload/chromewebstore/v1.1/items/$ITEM_ID
 Type: PUT
 Header Parameters: $TOKEN: the access token
```

Comment:
This format is confusing me. For all of the other items the format is basically "key: description", but here it reads like "key: key: description" or "key: header: description". I'm confused, though, because "$TOKEN" is not a header and replacing $TOKEN with the placeholder value also wouldn't be a valid header.

I think it would be best to change this line to be a bit clearer to the reader. I'd be happy to kick around ideas if you like.

Reply:
@dotproto I'm open to figuring out a better approach. Since you have more experience with API docs, I'm interested to hear your thoughts.